crates/cdk/src/nuts/nut04.rs

```diff
@@ -214,6 +214,8 @@ pub struct MintMethodSettings {
     /// Max Amount
     #[serde(skip_serializing_if = "Option::is_none")]
     pub max_amount: Option<Amount>,
+    /// Quote Description
+    pub quote_description: bool,
```

Comment:
Shouldn't this field be just called `description`?

The example here just shows `"description": true` - https://github.com/cashubtc/nuts/blob/main/04.md#settings